apps/kilocode-docs/docs/teams/billing.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebar_label: Billing & Credits
+---
+
+# Billing & Credits
+
+Kilo for Teams uses a transparent, two-part billing system: a monthly subscription for seats plus pay-as-you-go AI credits with zero markup.
+
+## Understanding the Billing Model
+
+### Monthly Subscription
+
+- **$29 per user per month** for platform access
+- **Billed monthly** on your signup anniversary
+- **Pro-rated** for mid-cycle seat changes
```

Comment:
pro-rated for seat additions. seat removals kick in at the start of the next month, with the seat still being active until the next billing cyc.e